debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/SourceInfo.java

```diff
@@ -17,12 +17,14 @@
 
     public static final String TXID_KEY = "txId";
     public static final String SCN_KEY = "scn";
+    public static final String SCN2_KEY = "scn2";
     public static final String COMMIT_SCN_KEY = "commit_scn";
+    public static final String COMMIT2_SCN_KEY = "commit2_scn";
     public static final String LCR_POSITION_KEY = "lcr_position";
     public static final String SNAPSHOT_KEY = "snapshot";
 
-    private long scn;
-    private Long commitScn;
+    private String scn;
```

Comment:
Can we use `Scn` here? Only when "rendering" this into the actual message `Struct` in `OracleSourceInfoStructMaker`, let's convert it to String (similar e.g. to the timestamp, which is an `Instant` for all purposes of internal processing and representation, but then is "rendered" as millis-since-epoch when creating the `Struct`.